### PR TITLE
(feat) Added day store and fake data generators

### PR DESCRIPTION
**Pull request by kyleaowen**
To review the fake data, start from the day-data-creator.js file and then follow the requires. There is a commented object which represents the data I have created. To have a look at the data, navigate to the dayData folder, enter the node interactive environment in the console, and type:

var d = require("./day-data-creator")
JSON.stringify(d.create(), null, 2);
take the resulting JSON object and through it in the console.

We should talk about the schema - this is just a draft, I assume we all have different ideas about how it should be structured.

### CODE REVIEW

File: client/js/dayData/day-data-creator.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/////////////////////////////////////////////////////////////////
+//   Hopefully we can continually modify this code as we change
+//   the schema, so we can always have sweet data generators!
+///////////////////////////////////////////////////////////////////
```
Comment (joshuaNewman10) [inline client/js/dayData/day-data-creator.js:4]: Is the idea to move these data creation files into our backend once we build that out this week?

File: client/js/stores/day-store.js [hunk 2]
```diff
@@ -1,9 +1,36 @@
 var Dispatcher = require('./dispatcher/dispatcher');
 var AppConstants = require('./constants/constants');
 var EventEmitter = require('event').EventEmitter;
+var assign = require('object-assign');
+var CHANGE_EVENT = 'change';
```
Comment (joshuaNewman10) [inline client/js/stores/day-store.js:5]: Shouldnt the CHANGE_EVENT be in our constants folder/file?

**Comment by joshuaNewman10** [comment]
This generally looks fantastic Kyle! Nice to have working data generators from such an early stage in development that we can modify. Just a few questions that we can discuss at tonights standup

File: client/js/stores/day-store.js (hunk 2 above)
Comment (kyleaowen) [inline client/js/stores/day-store.js:5]: glad we discussed this! we'll move it into the constants file if we deem it necessary

File: client/js/dayData/day-data-creator.js (hunk 1 above)
Comment (kyleaowen) [inline client/js/dayData/day-data-creator.js:4]: possibly! It will give us the option if we want

**Pull request merged**